libs/application/ui-fields/src/lib/FindVehicleFormField/FindVehicleFormField.tsx

```diff
@@ -305,6 +357,55 @@ export const FindVehicleFormField: FC<React.PropsWithChildren<Props>> = ({
                 focused={true}
               />
             )}
+            {energyDetails && !vehicleNotFound && (
+              <ActionCard
+                heading={`${energyDetails.make ?? ''} - ${
+                  energyDetails.permno
+                }`}
+                text={`${energyDetails.color} - ${
+                  energyFundsMessages &&
+                  formatText(
+                    energyFundsMessages.registrationDate,
+                    application,
+                    formatMessage,
+                  )
```

Comment:
If `energyFundsMessages` is false, won't the text end up as "color - false"?
I would rather use a ternary operator and fallback to empty string

```ts
energyFundsMessages ? formatText(....) : ''
```

Reply:
Yes, you are correct. Will fix